Sources/Valet/SecureEnclaveValet.swift

```diff
@@ -172,7 +172,27 @@ public final class SecureEnclaveValet: NSObject {
             try SecureEnclave.string(forKey: key, withPrompt: userPrompt, options: baseKeychainQuery)
         }
     }
-    
+
+    /// Attempts to retrieve the string at the given key, allowing a custom `fallbackTitle` to be specified to be
+    /// shown to the user in the case that the biometric authentication fails.
+    ///
+    /// - Parameters:
+    ///   - key: A key used to retrieve the desired object from the keychain.
+    ///   - userPrompt: The prompt displayed to the user in Apple's Face ID, Touch ID, or passcode entry UI.
+    ///   - fallbackTitle:  The title of the fallback button shown to the user after 2 failed retrieval attempts.
```

Comment:
I know I proposed this `fallbackTitle`, but looking at it in context I wonder if we can do better. The button this title text populates allows for cancelling the keychain lookup without cancelling the underlying action.

So it does provide a fallback option. But maybe we could describe the intent better? Maybe `alternativeOptionTitle`? Very open to suggestions here. I'm also okay with this name landing as-is if we can't agree on something better.

Reply:
Using "fallback" matches the prior art from `LAContext`, which might be nice for people that are transitioning over to Valet from a custom implementation.

Reply:
Fair that it may not be best to design this API in a vacuum. I can drop this one.